code/modules/events/_event.dm

```diff
@@ -31,8 +31,10 @@
 	var/auto_add = TRUE				//Auto add to the event pool, if not you have to do it yourself!
 	var/can_malf_fake_alert = FALSE	//Can be faked by malf ai?
 
-
-	var/dynamic_should_hijack = FALSE	// Whether or not dynamic should hijack this event
+	/// Whether or not dynamic should hijack this event to spawn its own events instead.
```

Comment:
a little inaccurate?

Should be like "Dynamic will block this event from spawning if ANY midround injection is about to happen within the next ~10 minutes"